src/cmd.rs

```diff
@@ -500,6 +500,102 @@ impl Command {
 
         self.cmd.spawn()
     }
+
+    /// Returns the path to the program that was given to [`Command::new`].
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```rust
+    /// use assert_cmd::Command;
+    ///
+    /// let cmd = Command::new("echo");
+    /// assert_eq!(cmd.get_program(), "echo");
```

Comment:
I tried replacing this with `Command::cargo_bin("bin_fixture")` to be more in-line with the other tests but it fails to run